### PR TITLE
scratch-messaging: make links an actual link

### LINKED ISSUE #2743: Copy link to comment in messaging tab
> Allow the "Copy link to comment" addon to be alsoin the messaging tab of Scratch Addons.

I'll stop my feedback issue rush now and continue tomorrow.

### PR DESCRIPTION
**Pull request by apple502j**
Resolves #2743
Resolves https://github.com/ScratchAddons/ScratchAddons/issues/799#issuecomment-731756883
Maybe can go into 1.16.2, no strings

### CODE REVIEW

File: popups/scratch-messaging/popup.html [hunk 1]
```diff
@@ -24,12 +24,14 @@
...
-            src="../../images/icons/popout.svg"
-            class="popout-comment"
-            @click="openComment()"
-            title="{{ messages.openNewTabMsg }}"
-          />
+          <a rel="noopener noreferrer" target="_blank" :href="commentURL">
+            <img
+              src="../../images/icons/popout.svg"
+              class="popout-comment"
+              @click="openComment()"
```
Comment (TheColaber) [inline popups/scratch-messaging/popup.html:31]: You still use openComment here?

Comment (apple502j) [reply popups/scratch-messaging/popup.html:31]: whoops

File: popups/scratch-messaging/popup.html [hunk 2]
```diff
@@ -224,7 +226,12 @@
         v-show="profile.unreadComments && profile.loadedComments"
       >
         <div class="message-type-title">
-          <a class="message-type-title-text" @click="openProfile(profile.username)">
+          <a
+            class="message-type-title-text"
+            rel="noreferrer noopener"
+            target="_blank"
+            href="https://scratch.mit.edu/users/{{ profile.username }}"
```
Comment (WorldLanguages) [inline popups/scratch-messaging/popup.html:233]: Maybe we should add trailing slash to avoid useless redirect?

**Comment by WorldLanguages** [comment]
@apple502j Please test before merging 😂

image

**Pull request merged**